Assets/HoloToolkit/Utilities/Scripts/Billboard.cs

```diff
@@ -24,8 +24,15 @@ public class Billboard : MonoBehaviour
         [Tooltip("Specifies the axis about which the object will rotate.")]
         public PivotAxis PivotAxis = PivotAxis.Free;
 
+        public Transform TargetTransform;
```

Comment:
Maybe we should add a tooltip noting that it gets directed to the mainCamera by default?